source/runtime.cpp

```diff
@@ -196,6 +195,8 @@ namespace reshade
 				}
 			}
 		}
+
+		_drawcalls = _vertices = 0;
```

Comment:
Any particular reason behind moving this line?

Reply:
The original position reset the values they received during the frame / in onpresent in the various hooked code and the statistics tab therefore showed them as '0' in v3.1 (as 'draw_overlay()' is after the original place in `runtime::on_present()`). The line is for resetting the values for the next frame after present, so moving the statement after `draw_overlay()` fixed that :)

Reply:
Oh. Good catch!